### ISSUE TITLE
IndexError:  array index out of range.

### ISSUE THREAD

**Issue by WakiTobaki** [kyledayton/HuionKamvasGT220v2LinuxDriver#1]
Upon running the driver in linux mint/ubuntu I receive the following error in terminal.

This had been working for me a month prior and I suppose something in one of the libraries has changed to invalidate the setting.

============================================
grabbed interface %d 0
grabbed interface %d 1
Huion Kamvas GT220 driver should now be running
Traceback (most recent call last):
  File "kamvas.py", line 45, in <module>
    X = (data[8] << 16) + (data[3] << 8) + data[2]
IndexError: array index out of range
============================================

Thanks in advance if someone sees this and can help me remedy it.

**Comment by kyledayton** [comment]
Did you recently update any packages? 
Could you try adding the following line above the existing line 45 in `kamvas.py`: 
```python
print("Data:", data)
```

Once you've done that, reply with the output

**Comment by WakiTobaki** [comment]
========================================
Huion Kamvas GT220 driver should now be running
Data: array('B', [7, 129, 115, 69, 130, 37, 17, 1])
Traceback (most recent call last):
  File "kamvas.py", line 46, in <module>
    X = (data[8] << 16) + (data[3] << 8) + data[2]
IndexError: array index out of range
========================================

I always thought python indexing started at 0, but this exact fix was working for me literally last week, so I'm not really aware of what I've done to mess it up.  I'm pretty new to linux, so it's easily possible that one of the required packages updated due to my own carelessness.

**Comment by WakiTobaki** [comment]
Hola!  I have resolved my issue and know roughly what went wrong.  I had compiled one of the necessary compenents under different circumstances from what was originally configured and forgot that I had done so and this is what caused the rest of the driver to error out.  Your code is still legitimate, I'll try to get specifics so other people don't hit the same speed bumps in the future.

Thanks for responding.
